openedx/stanford/djangoapps/auth_lagunita/tests/test_models.py

```diff
@@ -16,19 +16,23 @@ def test_basic(self):
     def test_create(self):
         seed_users(length=LENGTH_USERS)
         infos = Info.objects.all()
+        infos = list(infos)
         self.assertEquals(len(infos), LENGTH_USERS)
 
     def test_need_subscribed_empty(self):
         seed_users(always_optin=False, length=LENGTH_USERS)
         infos = Info.need_subscribed()
+        infos = list(infos)
         self.assertEquals(len(infos), 0)
 
     def test_need_subscribed_all(self):
         seed_users(always_optin=True, always_real_email=True, length=LENGTH_USERS)
         infos = Info.need_subscribed()
+        infos = list(infos)
         self.assertEquals(len(infos), LENGTH_USERS)
 
     def test_need_subscribed_some(self):
         seed_users(length=LENGTH_USERS)
         infos = Info.need_subscribed()
+        infos = list(infos)
```

Comment:
This is needed since `need_subscribed` now returns a generator, instead of an actual list.